x-pack/plugins/synthetics/public/legacy_uptime/components/monitor_management/monitor_list/actions.tsx

```diff
@@ -81,7 +81,7 @@ export const Actions = ({ id, name, onUpdate, isDisabled, errorSummaries, monito
             onUpdate={onUpdate}
             name={name}
             id={id}
-            isDisabled={isDisabled || isProjectMonitor || !canUpdatePrivateMonitor}
+            isDisabled={isDisabled || !canUpdatePrivateMonitor}
```

Comment:
I don't think we can get rid of this, without @drewpost 's sign off.

Reply:
this is only for testing purpose , wasn't meant to be pushed

Reply:
now this it has been, i wonder if we should keep this